StRoot/StFcsDbMaker/StFcsDbMaker.h

```diff
@@ -107,6 +108,7 @@ class StFcsDbMaker : public StMaker {
 private:
   StFcsDb *mFcsDb;
   int mDbAccess=1;
+  StFcsDbPulse* mFcsDbPulse;
```

Comment:
Technically, you need to increase the version in ClassDef when you change the class layout. But this is a maker, typically makers don't utilize streamers, so you can fix this class:

```diff
-  ClassDef(StFcsDbMaker,4)   //StAF chain virtual base class for Makers        
+  ClassDef(StFcsDbMaker,0)   //StAF chain virtual base class for Makers        
```

Reply:
done